odoo/models.py

```diff
@@ -5958,7 +5972,7 @@ def _get_redirect_suggested_company(self):
     #  - the global cache is only an index to "resolve" a record 'id'.
     #
 
-    def __init__(self, env: api.Environment, ids: tuple[int | NewId], prefetch_ids: Iterable[int | NewId]):
+    def __init__(self, env: api.Environment, ids: tuple[int | NewId] = (), prefetch_ids: Reversible[int | NewId] = ()):
```

Comment:
All three parameters are required on purpose.  Please don't change that.
```suggestion
    def __init__(self, env: api.Environment, ids: tuple[int | NewId], prefetch_ids: Reversible[int | NewId]):
```